pynitrokey/cli/nk3/__init__.py

```diff
@@ -506,7 +506,7 @@ def version(ctx: Context) -> None:
 @nk3.command()
 @click.pass_obj
 def factory_reset(ctx: Context) -> None:
```

Comment:
As discussed, we should add an `--experimental` flag to both commands.

Reply:
done